src/mdbExtensionController.ts

```diff
@@ -210,6 +211,36 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  registerUriHandler = (): void => {
+    vscode.window.registerUriHandler({
+      handleUri: async (uri: vscode.Uri): Promise<void> => {
+        const command = uri.path.replace(/^\//, '');
+        const parameters = uri.query.split('&').reduce((acc, param) => {
+          const [key, value] = param.split('=');
+          acc[key] = value;
+          return acc;
+        }, {} as Record<string, string>);
```

Comment:
Can we just use `URLSearchParams` for this? Otherwise you'll also need to account for percent-decoding, `=` in parameter values, `__proto__` as a parameter name, etc.

(Also: Object.fromEntries() is a thing now 🙂)